### DISCUSSION TITLE
Change the license of Spook?

### DISCUSSION THREAD

**Discussion by frenck** [frenck/spook#613] [Polls]
Spook will turn 1-year old on [23 February 2024](https://github.com/frenck/spook/commit/67803faf19bca7c8543e0865e1dba58755315652).

I originally started out to create something to experiment with (and I still do experiment concepts with Spook). However, nowadays, Spook seems to be gaining popularity and slowly getting an actual "fan base" (sounds weird to write/say).

I added the passive aggressive license to Spook. The main reason: it is a custom integration; custom integrations should be avoided as much as possible. Additionally, there is stuff in Spook that might not be the best idea.

The current passive aggressive license always seems to lead to questions and discussions, and more often seems to scare people.

Spook 👻 is turning one year old. How it developed and was embraced by part of the community, made me wonder: Is it time to open it up more and change the license?

If you vote, could you please tell me in the comments why you voted for that one? Thanks!

---

Related poll: [👻 Not your homie? Or your homie afterall?](https://github.com/frenck/spook/discussions/612)

**Comment by boheme61** [comment]
Id say keep it, it's pretty obvious that some people need to have things "spelled" out in a harsh way, but ofcause others could/would be scared , or annoyed ( However, that in the end just lead them into a more "be careful" mindset , which is good for everyone )

OK, i just read the (Your) License and the CCO License ... Mind-blowing :) LOL

**Comment by parautenbach** [comment]
I like the humour with the seriousness of the current license.

**Comment by Mariusthvdb** [comment]
yep. like the Humour too.. though Im told it actually isnt ;-)

Aging comes with being more serious though (purposely avoiding the responsibility concept here), so why not reflect that in the License too.

In the end, it doesn't seem to keep people from using Spook anyways..

**Comment by frenck** [comment]
I've decided to switch Spook's license to an OSI-approved open-source license.

Changed the license to MIT for v3+ of Spook:

<https://github.com/frenck/spook/pull/672>

../Frenck
